tests/options/pugExplicitDiv/unformatted.pug

```diff
@@ -0,0 +1,12 @@
+div.class(id="id" attribute="value") div with id attr test
```

Comment:
Maybe you can get rid of this file and rename the other two
Then you can use the always against the never case and the never case against the always 🤔

Only thing that wouldn't be tested then would be mixed -> formatted result 🤔

---

I think I have some similar tests, could you search a little bit and adept to a similar solution?
It's a little bit hard for me, cause I'm just on my phone right now 😅
Will be fully back from vacation late evening on next friday

Reply:
Okay, I'll look around and see what else I find that you have.  I was mostly following what I found in `pugClassNotation` that had:
* unformatted.pug
* formatted-attribute.pug
* formatted-literal.pug

I did it because I found in our code base a mixed set (someone managed to commit with no leading `div` and no one noticed).  I can see how the attribute and literal could be used to test each other though.  For development of the tests it was helpful to have the source and two targets so I could find where problems where.

You shouldn't review code while on vacation 🌴 - you should relax.  This can wait until you're back.